easybuild/easyconfigs/b/bitsandbytes/bitsandbytes-0.43.3-foss-2023a.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'CMakeMake'
+
+name = 'bitsandbytes'
+version = '0.43.3'
+
+homepage = 'https://huggingface.co/docs/bitsandbytes/main/en/index'
+description = "bitsandbytes enables accessible large language models via k-bit quantization for PyTorch."
+github_account = 'bitsandbytes-foundation'
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['7a468bc977da19c176cc578954bfd7a3c64182f387a6849e9f0a38d5cba1b4df']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('pkgconf', '1.9.5'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('PyTorch', '2.1.2'),
+    ('SciPy-bundle', '2023.07'),
+]
+
+skipsteps = ['install']
+
+postinstallcmds = [
+    'pip install --prefix=%(installdir)s --no-deps --ignore-installed --no-index --no-build-isolation %(start_dir)s',
+]
+
+sanity_check_paths = {
+    'files': ['lib/python3.11/site-packages/bitsandbytes/libbitsandbytes_cuda121.so'],
```

Comment:
```suggestion
    'files': ['lib/python%%(pyshortver)s/site-packages/bitsandbytes/libbitsandbytes_cuda121.%s' % SHLIB_EXT],
```

Reply:
a bit weird that the non-cuda version installs a cuda lib..

Reply:
Right. it should be cpu version

Reply:
It seems my test passed because I did not clean the failed installation properly.